src/main/java/com/qaprosoft/carina/demo/gui/components/FooterMenu.java

```diff
@@ -34,7 +34,11 @@
     
     @FindBy(linkText = "News")
     private ExtendedWebElement newsLink;
-
+    
+    private String str;
+    
+    private Integer i;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "i" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube:9000/sonarqube/coding_rules#rule_key=squid%3AS1068)